Refactor homepage. (#116)
ipfs/ipfs-docs-v2: docs/README.md

description: The home page for developer documentation for IPFS, the InterPlanetary File System.
---

> # Educate yourself

Author: I see what you mean, _Educate yourself_ might have been a bit too aggressive. I'm happy to go with your suggestion for now.

> <p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>

Author: I like all of this, except for the part about _centralization_. It's a bit of a weird word that's only really used in the blockchain/crypto industry.

More than happy to follow your guidance regarding font-weights.

> <p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>

Reviewer: Let’s chat at the docs weekly in a few?

> <p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>

Reviewer: Let’s chat at the docs weekly in a few?

> <p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>

Reviewer: Removed the "centralization" bit.

> ## Host a static website on IPFS

Reviewer: ```suggestion
## Host your website on IPFS
```
"Your" makes it a little more personal. We may want to downplay the "static" aspect in the header, too.

> Protect your site from a single point of failure by importing it to the IPFS network, and benefit from decentralized infrastructure. If you're new to IPFS, this is an excellent place to start learning how to build on the decentralized web. [Start the tutorial →](/how-to/host-single-page-site/)

Reviewer: ```suggestion
Protect yourself from single-point-of-failure risks and benefit from decentralized infrastructure by hosting your static website using IPFS. If you're new to IPFS, this is an excellent place to start learning how to build on the decentralized web. 

[Start the tutorial →](/how-to/host-single-page-site/)
```
Tightened the language a little and moved the arrow link to its own line for consistency with others. (Note: "IPFS network" is kind of squishy language just in terms of vaguely implying centralization in the network itself ... we may want to avoid that phrase.)

> ## Community projects

Reviewer: ```suggestion
## See what others are building
```
Reframing with some action language.

> Are you looking for inspiration? You can find a huge collection of awesome community projects over at Awesome IPFS. [Go there now →](https://awesome.ipfs.io/)

Reviewer: ```suggestion
Are you looking for inspiration? You can find a huge collection of awesome community projects over at Awesome IPFS. 

[Check out Awesome IPFS now →](https://awesome.ipfs.io/)
```
Moved arrow-link to its own line.

![The Awesome IPFS homepage.](./images/awesome-ipfs.png)

> ## Useful tools

Reviewer: ```suggestion
## Download IPFS tools
```
Reframing with some action language. I'd also strongly advocate moving this section to the top of the list, since it's what we want users to do the most - install Companion, Desktop, etc.

Author: I'm not sure whether it'd be best to have _Download IPFS tools_ or _Host a static website on IPFS_. Happy to put tools at the top for now, probably a good avenue for some A/B testing in the future though!

> Build your apps and services quicker with some help from these tools built for IPFS.

Reviewer: ```suggestion
These tools can help you use and build on IPFS more quickly and efficiently — give them a try today!
```

We may want to tweak this section a little bit sometime later, but this is a good start. However, let's not limit these to folks who are just developing (hence the change in language) ... people who just want to share files or play with the network can benefit from companion or desktop, too.

| [Browser Companion](https://github.com/ipfs-shipyard/ipfs-companion)   | [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop)| [IPLD Explorer](https://explore.ipld.io/) |
| --- | --- | --- |
| ![The IFPS browser companion in Firefox.](./images/ipfs-companion.png) | ![The IPFS desktop app running on MacOS with the status tab open.](./images/ipfs-desktop.png) | ![The IPLD Explorer homepage.](./images/ipld-explorer.png) |

> ## Decentralized web

Reviewer: ```suggestion
## Learn about the decentralized web
```
Reframing with some action language.

> Want to know more about the decentralized web (Dweb) and how it's changing the internet? These resources will give you a clearer understanding of the Web movement, and how it's improving the world:

Reviewer: ```suggestion
Want to know more about the decentralized web (Dweb) and how it's changing the internet? These resources will give you a clearer understanding of the Dweb movement, and how it's improving the world:
```
... right?

Author: Oooph, you caught my typo. Nice catch!

- [What is IPFS](/concepts/what-is-ipfs/)
- [Peer-to-peer sharing](/concepts/dht/)

> - [Content-addressed data](/concepts/content-addressing/)

Reviewer: ```suggestion
- [Content-addressed data](/concepts/content-addressing/)

[See more guides in the Concepts section →](/concepts/)
```
Added an arrow-link to the whole section for consistency with other parts of this page

> ## Community

Reviewer: ```suggestion
## Be part of the IPFS community
```
Reframing with some action language.

> IPFS has a bustling community of designers, developers, writers, and activists who are all helping to improve the project. You can join in by attending a local meetup, helping out at a conference, or chatting online through the `#ipfs` IRC channel, or on [the IPFS forum](https://discuss.ipfs.io/).

Reviewer: ```suggestion
IPFS has a bustling community of designers, developers, writers, and activists who are all helping to improve the project. You can join in by attending a local meetup, helping out at a conference, chatting online through the `#ipfs` [IRC channel](/community/irc), or joining a discussion in [the IPFS forum](https://discuss.ipfs.io/).
```
Direct link to IRC info page; disambiguated list a little bit

description: The home page for developer documentation for IPFS, the InterPlanetary File System.
---

# Learn how to build the future of the internet

> <p style="font-size:1.65rem; font-weight: 200; line-height: 2rem; margin-top: -15px;">No matter who you are or what you're creating, you can be part of the movement to create the next generation of the internet. This documentation site contains everything you need to start building using the InterPlanetary File System (IPFS) and pave the way for the future Web. Check out some of the most popular content in these docs below — or explore the main site menu to learn about key <a href="/concepts/">IPFS concepts</a>, try out <a href="/how-to/">how-to guides</a>, see <a href="/reference/">API and CLI references</a>, and more. Just want to <a href="/install/">install IPFS? We've got you covered.</a></p>

Reviewer: ```suggestion
<p style="font-size:1.65rem; font-weight: 200; line-height: 2rem; margin-top: -15px;">No matter who you are or what you're creating, you can be part of the movement to create the next generation of the internet. This site contains everything you need to start building using the InterPlanetary File System (IPFS) and pave the way for the future Web. Check out some of the most popular content in these docs below — or explore the main site menu to learn about <a href="/concepts/">key concepts</a>, try out <a href="/how-to/">how-to guides</a>, see <a href="/reference/">API and CLI references</a>, or just <a href="/install/">install IPFS</a> and get hacking.</p>
```

Trimmed a tiny bit of fat 😊
